code/clue.js

```diff
@@ -9,51 +9,179 @@ const mrGreen = {
   image: 'assets/green.png',
   occupation: 'Entrepreneur'
 }
+  const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'violet',
+  description: 'Loves ice cream',
+  age: 45,
+  image: 'assets/plum.png',
+  occupation: 'Professor'
+}
+  const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She knows a lot about birds',
+  age: 45,
+  image: 'assets/scarlet.png',
+  occupation: 'Doktor'
+}
+  const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She went to Technigo',
+  age: 45,
+  image: 'assets/peacock.png',
+  occupation: 'Frontend developer'
+}
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'He loves custard',
+  age: 45,
+  image: 'assets/mustard.png',
+  occupation: 'Chef'
+}
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'She knows karate',
+  age: 45,
+  image: 'assets/white.png',
+  occupation: 'Headmaster'
+}
+
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+// CREATED OBJECTS FOR ALL THE WEAPONS
 
-const rope = {
-  name: 'rope',
+const Rope = {
+  name: 'Rope',
   weight: 10
 }
+const Knife = {
+  name: 'Knife',
+  weight: 1
+}
+const Candlestick = {
+  name: 'Candlestick',
+  weight: 15
+}
+const Dumbbell = {
+  name: 'Dumbbell',
+  weight: 30
+}
+const Poison = {
+  name: 'Poison',
+  weight: 3
+}
+const Axe = {
+  name: 'Axe',
+  weight: 16
+}
+const Bat = {
+  name: 'Bat',
+  weight: 9
+}
+const Trophy = {
+  name: 'Trophy',
+  weight: 50
+}
+const Pistol = {
+  name: 'Pistol',
+  weight: 8
+}
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
-
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
-
+// SUSPECTS
 const suspects = [
-  mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrGreen, mrsWhite, professorPlum, missScarlet, mrsPeacock, colonelMustard, 
 ]
 
-const weapons = []
+// WEAPONS 
+const weapons = [
+   Rope, Knife, Candlestick, Dumbbell, Poison, Axe, Bat, Trophy, Pistol
+]
 
```

Comment:
You chose to put your suspects and weapons arrays on one line - I chose to put them on separate lines. Your approach means fewer lines of code, and my approach is perhaps more readable. Don't know which is to prefer - just sayin' 🙃